x-pack/plugins/maps/common/ems_util.test.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import {
+  getEMSResources
+} from './ems_util';
+
+// eslint-disable-next-line import/no-unresolved
+import { getEMSClient } from 'ui/vis/__tests__/map/ems_client_util.js';
+
+describe('ems util test', () => {
+
+
+  it('Should get relative paths when using proxy', async () => {
+
+    const emsClient = getEMSClient({});
+    const isEmsEnabled = true;
+    const licenseId = 'foobar';
+    const useProxy = true;
+    const resources = await getEMSResources(emsClient, isEmsEnabled, licenseId, !useProxy);
```

Comment:
`getEMSResources` signature lists last parameter as `useAbsoluteEMSPath` but you are calling it `useProxy` in the tests and `proxyElasticMapsServiceInMaps` in the wild. Maybe  `getEMSResources` should rename last parameter to `useProxy` to be consistent with the common usage pattern